bcl/src/bc_module_infra_grid.c

```diff
@@ -285,7 +350,19 @@ static void _bc_module_infra_grid_task_measure(void *param)
             if (self->_enable_sleep)
             {
                 // Sleep Mode
-                bc_i2c_memory_write_8b(self->_i2c_channel, self->_i2c_address, _BC_AMG88xx_PCLT, 0x10);
+                if (self->_revision == BC_MODULE_INFRA_GRID_REVISION_R1_0)
+                {
+                    // Sleep mode
+                    if (!bc_i2c_memory_write_8b(self->_i2c_channel, self->_i2c_address, _BC_AMG88xx_PCLT, 0x10))
+                    {
+                        goto start;
+                    }
+                }
+                else if (self->_revision == BC_MODULE_INFRA_GRID_REVISION_R1_1)
+                {
+                    // Revision 1.1 has power disconnect
```

Comment:
I suggest to remove such comment and `#define _BC_MODULE_INFRA_GRID_PIN_POWER BC_TCA9534A_PIN_P7`